client/tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "compilerOptions": {
         "target": "es2016",
         "module": "es2020",
+        "verbatimModuleSyntax": false,
```

Comment:
Since we're overriding the recommended vue default here, could we add a comment as to why for future reference?